### PR TITLE
Modify table of contents, header and footer

### PR DESCRIPTION
**Pull request by chses9440611**
If the  header and footer still break table of contents[TOC], please tell me what broken TOC looks like.

### CODE REVIEW

**Comment by jserv** [comment]
Use `git rebase` rather than `git merge` before submitting.

File: lkmpg.tex [hunk 1]
```diff
@@ -3,7 +3,17 @@
...
 \usepackage{graphicx}
+\usepackage{tocloft}
+
+% fancyhdr settings
 \usepackage{fancyhdr}
+\fancyhf{}
+\pagestyle{fancy}
+\lhead{\rightmark}
+\rhead{\thepage}
+\cfoot{The Linux Kernel Module Programming Guide}
```
Comment (jserv) [inline lkmpg.tex:14]: The string "The Linux Kernel Module Programming Guide" would appear in TeX script. Please revise.

**Review by jserv — CHANGES_REQUESTED** [review]
Improve the subject of pull request and git commits. You should summarize the motivation and considerations.

**Comment by jserv** [comment]
Did you improve the subject and make use of `git rebase`?

**Comment by chses9440611** [comment]
> Did you improve the subject and make use of `git rebase`?

Not yet, still try to modify the commits with rebase.

**Comment by chses9440611** [comment]
@jserv  I didn't "make" twice at first, and mistook the book without no table of contents. Therefore the modification I made was trivial. Could I  delete this PR?

**Pull request closed without merging**